src/Microsoft.Extensions.DependencyModel/Resolution/PackageCacheCompilationAssemblyResolver.cs

```diff
@@ -42,37 +41,19 @@ public bool TryResolveAssemblyPaths(CompilationLibrary library, List<string> ass
 
             if (!string.IsNullOrEmpty(_packageCacheDirectory))
             {
-                var hashSplitterPos = library.Hash.IndexOf('-');
```

Comment:
Looking more at that change, this environment variable went away completely.  :(